guide/src/building-and-distribution.md

```diff
@@ -92,8 +92,9 @@ If you're packaging your library for redistribution, you should indicated the Py
 
 To use PyO3 with bazel one needs to manually configure PyO3, PyO3-ffi and PyO3-macros. In particular, one needs to make sure that it is compiled with the right python flags for the version you intend to use.
 For example see:
-1. https://github.com/OliverFM/pytorch_with_gazelle -- for a minimal example of a repo that can use PyO3, PyTorch and Gazelle to generate python Build files.
-2. https://github.com/TheButlah/rules_pyo3 -- which has more extensive support, but is outdated.
+1. https://github.com/abrisco/pyo3
```

Comment:
Did you want to link to `/pyo3` or `/rules_pyo3`?

Also, it would be good to add a description after `--` like for the others.

Reply:
Oops! Updated and added!